validator/src/main/jkube/deployment.yml

```diff
@@ -2,18 +2,26 @@ spec:
   template:
     spec:
       containers:
-      - env:
-          - name: JAVA_OPTIONS
-            value: " -Dgreeting=@greeting@ "
-      livenessProbe:
-        httpGet:
-          path: /
-          port: 8161
-          scheme: HTTP
-        initialDelaySeconds: 180
-      readinessProbe:
-        httpGet:
-          path: /
-          port: 8161
-          scheme: HTTP
-        initialDelaySeconds: 10
\ No newline at end of file
+        - env:
+            - name: JAVA_OPTIONS
+              value: " -Dgreeting=@greeting@ "
+          livenessProbe:
+            httpGet:
+              path: /actuator/health/liveness
+              port: 8081
+              scheme: HTTP
+            initialDelaySeconds: 180
+          readinessProbe:
+            httpGet:
+              path: /actuator/health/readiness
+              port: 8081
+              scheme: HTTP
+            initialDelaySeconds: 10
+          lifecycle:
+            postStart:
+              exec:
+                command: [ "/bin/sh", "-c", "sleep 1 ; echo In postStart > /dev/termination-log" ]
+            preStop:
+              exec:
+                command: [ "/bin/sh", "-c", "sleep 1 ; echo In preStop > /dev/termination-log" ]
+      terminationGracePeriodSeconds: 60 # default is 30
```

Comment:
could you mention this property and the camel.main.shutdown-timeout in the README?

something like `Given that by default terminationGracePeriodSeconds is 30 seconds and camel.main.shutdown-timeout is 45, this example shows how to configure both of them so that the pod is not killed before the shutdown-timeout`.

Though, the terminationGracePeriod or the camel shutdown timeout does not apply that much to this example, what do you think adding showing these properties to an example that deal with queues?

Reply:
I think there would be no need to add a more complex example since the right behaviour will be logged.

Surely we can mark this PR as a draft, till the README file is updated properly